bower.json

```diff
@@ -21,18 +21,36 @@
   "homepage": "https://github.com/polymerelements/app-localize-behavior",
   "ignore": [],
   "dependencies": {
-    "iron-ajax": "PolymerElements/iron-ajax#^1.0.0",
+    "iron-ajax": "PolymerElements/iron-ajax#2.0-preview",
     "intl-messageformat": "yahoo/intl-messageformat#^1.0.0",
-    "polymer": "Polymer/polymer#^1.3.0"
+    "polymer": "polymer/polymer#^2.0.0-rc.1"
   },
   "devDependencies": {
     "intl": "andyearnshaw/intl.js#^1.0.0",
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-    "paper-toggle-button": "PolymerElements/paper-toggle-button#^1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "iron-component-page": "PolymerElements/iron-component-page#2.0-preview",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
+    "paper-toggle-button": "PolymerElements/paper-toggle-button#2.0-preview",
+    "paper-styles": "PolymerElements/paper-styles#2.0-preview",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.6",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1"
   },
-  "main": "app-localize-behavior.html"
+  "main": "app-localize-behavior.html",
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "iron-ajax": "PolymerElements/iron-ajax#^1.0.0",
+        "polymer": "Polymer/polymer#^1.1.0"
```

Comment:
We discussed this offline but we need to change this to `^1.7.0` or maybe `^1.8.0` (here and in all of the other hybrid elements).